[Quick starts] Correct add_hypertable example rails migration and remove incorrect references to time column (#1407)
timescale/docs: timescaledb/quick-start/ruby.md

In your `db/migrate` project folder, you'll see a new migration file for
`[some sequence of numbers]_add_hypertable`.

> Then we can write the migration to add our hypertable like so:

Reviewer: ```suggestion
Then you can write the migration to add the hypertable:
```
